workers/unity/Assets/Playground/Config/ArchetypeConfig.cs

```diff
@@ -8,6 +9,7 @@ public static class ArchetypeConfig
     {
         public const string CharacterArchetype = "Character";
         public const string CubeArchetype = "Cube";
+        public const string PlayerCreator = "PlayerCreator";
```

Comment:
This doesn't belong here I don't think.

There aren't any special components we put on the PlayerCreator